theme/sass/components/_hero-image.scss

```diff
@@ -11,12 +11,24 @@
         background-image: url("../img/teaching-image.jpeg");
     }
 
-    .hero-text-box {
+    .hero-text {
         position: absolute;
-        left: 10vw;
-        bottom: 10vw;
+        left: 5vw;
+        bottom: 5vw;
+        max-width: 15rem;
         text-align: left;
         font-weight: bold;
         color: white;
+        padding: 0 2rem;
+        border-radius: 10px;
+        background-color: transparentize($secondary-color, 0.7);
+
+        @include breakpoint($md) {
+            max-width: 200rem;
+            position: absolute;
```

Comment:
Remove position: absolute from here its not doing anything.

Reply:
done